vhd/packer/configure-windows-vhd.ps1

```diff
@@ -49,8 +49,11 @@ function Get-ContainerImages {
                 "mcr.microsoft.com/windows/nanoserver:1809",
                 "mcr.microsoft.com/oss/kubernetes/pause:1.4.1",
                 "mcr.microsoft.com/oss/kubernetes/pause:3.4.1",
-                "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.0.1-alpha.1-windows-1809-amd64",
-                "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v1.2.1-alpha.1-windows-1809-amd64")
+                "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.0.8",
+                "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.1.4",
+                "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.10.0",

```

Comment:
Are these 3 CSI images Windows container images?

Reply:
Yes - https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/deploy/csi-azuredisk-node-windows.yaml

Reply:
The liveness probe and node-driver-registrar containers are shared by all CSI plugin.

Reply:
In AKSe, we have different image reference for Linux and Windows pointing to the same image (https://github.com/Azure/aks-engine/blob/master/pkg/api/k8s_versions.go#L39-L42), and I have checked pods on windows nodes, and they are using these images.